packages/react-email-templates/src/index.ts

```diff
@@ -1,13 +1,28 @@
 import { render } from '@react-email/render'
 import { APIGatewayEvent } from 'aws-lambda'
 import {
+	ErrorAlertEmail,
+	LogAlertEmail,
+	NewSessionAlertEmail,
+	NewUserAlertEmail,
+	RageClickAlertEmail,
 	SessionInsightsEmail,
-	SessionInsightsEmailProps,
-} from '../emails/session-insights'
+	TrackEventPropertiesAlertEmail,
+	TrackUserPropertiesAlertEmail,
+} from '../emails'
 
 export const handler = async (event?: APIGatewayEvent) => {
-	const args = JSON.parse(event?.body ?? '{}') as SessionInsightsEmailProps
-	const html = render(SessionInsightsEmail(args))
+	const args = JSON.parse(event?.body ?? '{}')
+
+	const EmailTemplate = getEmailTemplate(args.template)
+	if (!EmailTemplate) {
+		return {
+			statusCode: 400,
+			body: 'Request is missing a valid "template" property',
+		}
+	}
+
+	const html = render(EmailTemplate(args.data))
```

Comment:
Breaking change - wondering if we should redeploy this as an entirely different lambda function since it is not longer specific to the insights email?

Reply:
Synced with Zane, and this shouldn't be an issue